### DISCUSSION TITLE
How to add ArkAscended

### DISCUSSION THREAD

**Discussion by Setman590** [tltneon/lgsl#185] [General]
Hi guys,

Trying to find a way to add ArkAscended to my server list

**Comment by tltneon** [comment]
Hi @Setman590 
I'd like to add but I can't find any docs about querying ArkAscended neither I don't have a game copy to explore packets.

**Comment by Setman590** [comment]
OK Thanks

**Comment by tltneon** [comment]
@Setman590 I added Ark Ascended support
